daemon/rpc_connect_test.go

```diff
@@ -165,84 +166,150 @@ func TestRpcConnect(t *testing.T) {
 			checker: &workingLoginChecker{vpnErr: errors.New("test error")},
 			resp:    internal.CodeTokenRenewError,
 		},
+		// TODO: add a test for DIP
```

Comment:
waiting for DIP PR to be merged https://github.com/NordSecurity/nordvpn-linux/pull/503